tests/garage/tf/models/test_model.py

```diff
@@ -191,6 +196,12 @@ def test_complicated_model_is_pickleable(self):
             input_var = tf.placeholder(tf.float32, shape=(None, 5))
             outputs = model.build(input_var)
 
+            # assign bias to all one
+            with tf.variable_scope(
+                    'ComplicatedModel/SimpleModel/state', reuse=True):
+                bias = tf.get_variable('hidden_0/bias')
+            sess.run(tf.assign(bias, tf.ones_like(bias)))
```

Comment:
i think you can use tf.Variable.load() directly instead of tf.assign()

See the pickler in Model

Reply:
Would that be a good idea if I create another PR to replace all the `tf.assign()` to `tf.Variable.load()`?